resources/views/livewire/user-list.blade.php

```diff
@@ -0,0 +1,104 @@
+<div class="p-5">
+
+    @if(session()->has('message'))
+        <div class="bg-green-100 border-l-4 border-green-500 text-green-700 p-4 mb-4">
+            <p>{{ session('message') }}</p>
+        </div>
+    @endif
+
+    <div class="flex justify-end items-center mb-4">
+        <div class="flex space-x-2">
+            <x-input
+                    type="text"
+                    placeholder="Search..."
+                    wire:model.debounce="search"
+            />
+        </div>
+    </div>
+
+    <div class="bg-white dark:bg-slate-800 shadow-md rounded-lg p-5 overflow-x-auto">
```

Comment:
@SerhiiCho would you want to take a look from the frontend point of view? We don't need to address issues in this PR necessarily, but it might be good trying to settle on reusable admin components.

Reply:
Sorry, but this PR has no description, I have no idea what is going on here

Reply:
It's addressing this issue: https://github.com/brendt/rfc-vote/issues/175

No worries if you don't have the time for it :)

Reply:
I have time, I just don't know how to pull this to my computer and push changes since it's separate origin

Reply:
Gotcha! Github actually shows you the commands:

<img width="815" alt="Screenshot 2023-09-06 at 20 43 11" src="https://github.com/brendt/rfc-vote/assets/6905297/2dbb96cc-fb6b-45b9-b727-b7d4fc973c35">

Press "Command line instructions":

<img width="815" alt="Screenshot 2023-09-06 at 20 43 17" src="https://github.com/brendt/rfc-vote/assets/6905297/6497da12-d36f-4159-89fb-f19d5fcef1ea">

Reply:
It's easier for you because you are the owner, for me, it's a bit tricky (i guess)